step.yml

```diff
@@ -54,42 +54,50 @@ inputs:
       is_required: true
   - commit: "$BITRISE_GIT_COMMIT"
     opts:
-      category: Clone Config
+      category: "Clone arguments"
       title: "Git Commit to clone"
       is_dont_change_value: true
   - tag: "$BITRISE_GIT_TAG"
     opts:
-      category: Clone Config
+      category: "Clone arguments"
       title: "Git Tag to clone"
       is_dont_change_value: true
   - branch: "$BITRISE_GIT_BRANCH"
     opts:
-      category: Clone Config
+      category: "Clone arguments"
       title: "Git Branch to clone"
       is_dont_change_value: true
   - branch_dest: "$BITRISEIO_GIT_BRANCH_DEST"
     opts:
-      category: Clone Config
+      category: "Clone arguments"
       title: "Destination git Branch, used for pull requests"
```

Comment:
Is there any reason for having Branch with a capital B? I'm just curious. If not, we can revert to lower case :)

Reply:
no there was none.